k8s/gcp/k8s-installer/create-k8s-cluster.yml

```diff
@@ -38,7 +38,7 @@
     - name: Creating the Cluster & InstanceGroup objects in our state store
       shell: |
         export KOPS_FEATURE_FLAGS=AlphaAllowGCE
-        kops create cluster {{ cluster_name }}.k8s.local --kubernetes-version {{ k8s_version }} --zones {{zone}} --state gs://{{ cluster_name }}/ --project {{ project }} --node-count {{ nodes }} --networking kubenet --image "ubuntu-os-cloud/ubuntu-1604-xenial-v20170202" --vpc {{ vpc }}
+        kops create cluster {{ cluster_name }}.k8s.local --kubernetes-version {{ k8s_version }} --zones {{zone}} --state gs://{{ cluster_name }}/ --project {{ project }} --node-count {{ nodes }} --networking kubenet --image "ubuntu-os-cloud/ubuntu-1604-xenial-v20170202" --vpc {{ vpc }} --node-size {{ node_size | default('n1-standard-4')}} --master-size {{ master_size | default('n1-standard-4')}}
     - name: Creating K8s Cluster
```

Comment:
Thanks for the changes @harshvkarn ! Was wondering, would it be ok to reduce downgrade the master instance size..?

Reply:
Great, thanks @harshvkarn !